prime-router/settings/STLTs/MS/ms-doh.yml

```diff
@@ -0,0 +1,108 @@
+# To submit this to localhost, run:
+#
+# ./prime multiple-settings set --input ./settings/STLTs/MS/ms-doh.yml
+---
+- name: "ms-doh"
+  description: "Mississippi Department of Health"
+  jurisdiction: "STATE"
+  stateCode: "MS"
+  countyName: null
+  filters: null
+  featureFlags: null
+  keys: null
+  senders: []
+  receivers:
+    - name: "full-elr"
+      organizationName: "ms-doh"
+      topic: "full-elr"
+      customerStatus: "active"
+      translation: !<HL7>
+        schemaName: "azure:/metadata/hl7_mapping/receivers/STLTs/MS/MS-receiver-transform.yml"
+        useTestProcessingMode: false
+        useBatchHeaders: true
+        receivingApplicationName: "MSDH-ELR"
+        receivingApplicationOID: null
+        receivingFacilityName: "MSDOH"
+        receivingFacilityOID: null
+        messageProfileId: null
+        replaceValue: {}
+        replaceValueAwithB:
+          MSH-6:
+            - '*': "MSDOH"
+          MSH-5:
+            - '*': "MSDH-ELR"
+          MSH-3:
+            - '*': "CDC PRIME - Atlanta, Georgia (Dekalb)^2.16.840.1.114222.4.1.237821^ISO"
+        reportingFacilityName: null
+        reportingFacilityId: null
+        reportingFacilityIdType: null
+        suppressQstForAoe: false
+        suppressHl7Fields: null
+        suppressAoe: false
+        defaultAoeToUnknown: false
+        replaceUnicodeWithAscii: false
+        useBlankInsteadOfUnknown: null
+        truncateHDNamespaceIds: false
+        truncateHl7Fields: null
+        usePid14ForPatientEmail: true
```

Comment:
You're missing the transform for this setting. Whenever a STLT has this setting on you can use this enrichment schema: 
```
    enrichmentSchemaNames:
      - "classpath:/metadata/fhir_transforms/common/use-pid-14-for-email.yml"
 ```